PSI_Tracker.js

```diff
@@ -0,0 +1,644 @@
+}
+
+
+/**
+ * Simple runBatch function to be called by trigger
+ */
+function runBatchSustainability() {
+  runBatch("Sustainability")
+}
+
+
+
+
+/**
+ * Parse URLs from the "Config" tab and sends it to the selected API
+ */
+function performTasks(api = "PSI", save_screenshot = false) {
+
+    // Show a warning that the data will be send to a 3P that might collect information
+    if(api == "Green Domain"){
+      if(showConfirmationModalCallGWF()){
+        // if permission declined, abort.
+        return true;
+      }
+    }
+
+    // Show a warning that the data will be send to a 3P that might collect information
+    if(api == "Sustainability"){
+      if(showConfirmationModalSustainability()){
+        // if permission declined, abort.
+        return true;
+      }
+    }
+
+    // Add the loading symbols and initialise a blank state
+    initURLStatus();
+    // Run batches
+    const execution_mode_boolean = CONFIG_SHEET.getRange("B24").getValue();
+    const execution_mode = execution_mode_boolean ? "TRIGGER" : "SERIAL";
+    // Default execution mode: queries run one after the other and might timoute
+    if(execution_mode == "SERIAL"){
+      while(runBatch(api, save_screenshot)){
+          Logger.log("Batch done")
+      }
+      toast("Done","Finished")
+    }
+    // Advanced mode with triggers
+    else if(execution_mode == "TRIGGER"){
+      removeBatchTriggers();
+      SpreadsheetApp.flush();
+      const TIME_BETWEEN_BATCHES = CONFIG_SHEET.getRange("B27").getValue();
+      if(api == "PSI" && save_screenshot == true){
+        ScriptApp.newTrigger('runBatchPSIWithScreenshots')
+          .timeBased()
+          .everyMinutes(TIME_BETWEEN_BATCHES)
+          .create();
+      }
+      else if(api == "PSI" && save_screenshot == false){
+        ScriptApp.newTrigger('runBatchPSI')
+          .timeBased()
+          .everyMinutes(TIME_BETWEEN_BATCHES)
+          .create();
+      }
+      else if(api == "CrUX"){
+        ScriptApp.newTrigger('runBatchCrUX')
+          .timeBased()
+          .everyMinutes(TIME_BETWEEN_BATCHES)
+          .create();
+      }
+      else if(api == "CrUX History"){
+        ScriptApp.newTrigger('runBatchCrUXHistory')
+          .timeBased()
+          .everyMinutes(TIME_BETWEEN_BATCHES)
+          .create();
+      }
+      else if(api == "Green Domain"){
+        ScriptApp.newTrigger('runBatchGreenDomain')
+          .timeBased()
+          .everyMinutes(TIME_BETWEEN_BATCHES)
+          .create();
+      }
+      else if(api == "Accessibility"){
+        ScriptApp.newTrigger('runBatchAccessibility')
+          .timeBased()
+          .everyMinutes(TIME_BETWEEN_BATCHES)
+          .create();
+      }
+      else if(api == "Sustainability"){
+        ScriptApp.newTrigger('runBatchSustainability')
+          .timeBased()
+          .everyMinutes(TIME_BETWEEN_BATCHES)
+          .create();
+      }
+      runBatch(api, save_screenshot);
+    }
+}
+
+
+const COLUMN_STATUS = 8;
```

Comment:
Maybe move to the top with the other global consts and add a comment on the magic number?

Reply:
done!